src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -859,10 +861,13 @@ public Node visitParenthesizedExpr(JsoniqParser.ParenthesizedExprContext ctx) {
 
     @Override
     public Node visitVarRef(JsoniqParser.VarRefContext ctx) {
-        String name = ctx.name.getText();
+        String localName = ctx.name.getText();
+        String prefix = null;
         if (ctx.ns != null) {
-            name = name + ":" + ctx.ns.getText();
+            prefix = ctx.ns.getText();
         }
+        // TODO resolved namespace
+        Name name = Name.createVariableInNoNamespace(localName);
```

Comment:
TODO & Prefix seems unused

Reply:
That's intentional, it will be used in a follow up PR when we have namespace declarations.